### ISSUE TITLE
Build a website to search for packages

### ISSUE THREAD

**Issue by hmsjy2017** [raspbian-addons/raspbian-addons#238]
I think we might be able to build a website like https://www.debian.org/distrib/packages to search for packages, which would make it easier for users to find packages. @ryanfortner 

Source code of the Debian Packages website is available: https://salsa.debian.org/webmaster-team/packages

**Comment by ryanfortner** [comment]
Good idea. I've forked the repository here: https://github.com/raspbian-addons/packages.raspbian-addons.org, feel free to experiment with it. I don't have time to set it up right now but I can try later.

**Comment by 89paradox** [comment]
I am currently working on a webpage to search for packages.

**Comment by ryanfortner** [comment]
any progress @hmsjy2017 ?

**Comment by hmsjy2017** [comment]
@ryanfortner I have tried but failed. Its document is outdated and lacks maintenance. In addition, I am currently busy with my studies and have no free time to maintain. I'm sorry.
